x-pack/plugins/telemetry/server/routes/get_stats.ts

```diff
@@ -0,0 +1,46 @@
+/*
```

Comment:
This is very similar to  `x-pack/plugins/telemetry/server/collectors/get_stats.ts` - any chance for reuse?

Reply:
Oh yes! The other one is not used.